tutorials/raw/plot_10_raw_overview.py

```diff
@@ -328,9 +328,9 @@
 # or combine two or more separate :class:`~mne.io.Raw` objects — you can use
 # the :meth:`~mne.io.Raw.append` method:
 
-raw_selection1 = raw.copy().crop(tmin=30, tmax=30.1)     # 0.1 seconds
-raw_selection2 = raw.copy().crop(tmin=40, tmax=41.1)     # 1.1 seconds
-raw_selection3 = raw.copy().crop(tmin=50, tmax=51.3)     # 1.3 seconds
+raw_selection1 = raw.copy().crop(tmin=30, tmax=30.1)  # 0.1 seconds
+raw_selection2 = raw.copy().crop(tmin=40, tmax=41.1)  # 1.1 seconds
+raw_selection3 = raw.copy().crop(tmin=50, tmax=51.3)  # 1.3 seconds
```

Comment:
idem.

Reply:
make sure that you don't touch files (or even file parts) that you are not modifying.

Reply:
Yes. This somehow was formatted strangely in my IDE, and didn't pass through flake.
I'll revert all that.